src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -56,10 +56,10 @@ public void OnResourceExecuting(ResourceExecutingContext context)
                 MediaTypeHeaderValue requestContentType = null;
                 MediaTypeHeaderValue.TryParse(context.HttpContext.Request.ContentType, out requestContentType);
 
-                // Only execute if this is the last filter before calling the action.
```

Comment:
It was a duplicate comment. Open your window on this diff just a bit wider, look at https://github.com/aspnet/Mvc/pull/3317/files#diff-1b5675f1b083c83b1f98dd3e87cf08f6R52